packages/rule-utils/src/themeFn.ts

```diff
@@ -0,0 +1,46 @@
+import MagicString from 'magic-string'
+import { colorToString, parseCssColor } from './colors'
+
+export const themeFnRE = /theme\(\s*['"]?(.*?)['"]?\s*\)/g
+
+export function hasThemeFn(str: string) {
+  return str.includes('theme(') && str.includes(')')
+}
+
+export function transformThemeFn(code: string, theme: Record<string, any>, throwOnMissing = true) {
+  const matches = Array.from(code.toString().matchAll(themeFnRE))
+
+  if (!matches.length)
+    return code
```

Comment:
Here we could return `false`

Reply:
I think we should keep the return `code`. Usually, we don't need to handle the return. If it return `false`, we need to handle the `false` case in every place where we use `transformThemeFn`